src/applications/covid-research-volunteer/pages/covidResearchUISchema.js

```diff
@@ -19,15 +19,19 @@ export const uiSchema = {
     'view:descriptionText': {
       'ui:description': (
         <span>
-          Thank you for your interest in volunteering for COVID-19 research at
-          VA. Please answer the questions below, and we’ll add you to our
-          volunteer registry. If we think you may be eligible for a study, we’ll
-          contact you to tell you more about it so you can decide if you want to
-          join. You don’t need to be a Veteran to volunteer.
+          Thank you for your interest in participating in coronavirus research
+          at VA. Please answer the questions below, and we’ll add you to our
+          volunteer registry. If we think you may be eligible for one of our
+          COVID-19 studies, we’ll contact you to tell you more about it so you
+          can decide if you want to join. You don’t need to be a Veteran to
+          volunteer.
           <p>
             <b>Note:</b> We won’t share your information with anyone outside of
-            VA. To learn more before volunteering, read about frequently asked
-            questions about COVID-19 research at VA.
+            VA. To learn more before volunteering, read about{' '}
+            <a href="https://staging.va.gov/alt-covid-research-lp/">
```

Comment:
```suggestion
            <a href="/alt-covid-research-lp">
```
use relative link so it is independent from environment